packages/verii-verification/src/verify-verii-credentials.js

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2025 Velocity Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const { initHttpClient } = require('@verii/http-client');
+const fetchers = require('@verii/common-fetchers');
+const { verifyCredentials } = require('./verify-credentials');
+
+const verifyVeriiCredentials = async (
+  { credentials, expectedHolderDid, relyingParty },
+  context
+) => {
+  const httpClient = initHttpClient({
+    nodeEnv: context.config.nodeEnv,
+    requestTimeout: context.config.requestTimeout,
+    traceIdHeader: context.config.traceIdHeader,
+    prefixUrls: [context.config.registrarUrl],
+    agent: context.agent,
+  });
+
+  // eslint-disable-next-line better-mutation/no-mutation
```

Comment:
This is a change relative to our "fetcher injection" architecture, do we want to allow injection of `registrarFetch` like we are allowing injection of `tenant` optionally

Reply:
Its an optional version that includes its own fetchers since that is more usable for third parties